quiz.css

```diff
@@ -0,0 +1,69 @@
+/*Block Styling and Positioning*/
+.medium,
+.dark,
+.tshirt,
+.green {
+  border:1px solid lightblue;
```

Comment:
Ah, yes. I had them with separate classes for some reason to start and when I was messing with things I combined them with the intention of going back and fixing that. Thanks!